AnwesenheitsSimulation/module.php

```diff
@@ -376,8 +378,10 @@ public function UpdateTargets()
 
         if (isset($NextSimulationData['nextSwitchTimestamp'])) {
             $this->SetTimerInterval('UpdateTargetsTimer', ($NextSimulationData['nextSwitchTimestamp'] - $this->getTime() + 1) * 1000);
+            $this->SetValue('NextTrigger', ($NextSimulationData['nextSwitchTimestamp'] - $this->getTime() + 1) * 1000);
```

Comment:
I think this should be $NextSimulationData['nextSwitchTimestamp'] only. Can you verify this? (Your current code shouldn't work properly!?)